airbyte-integrations/connectors/source-google-search-console/source_google_search_console/source.py

```diff
@@ -79,14 +81,17 @@ def _validate_and_transform(self, config: Mapping[str, Any]):
                     if dimension not in SearchAnalyticsByCustomDimensions.dimension_to_property_schema_map:
                         raise Exception(f"dimension: '{dimension}' not found")
 
-        pendulum.parse(config["start_date"])
+        # start date checks
+        pendulum.parse(config.get("start_date") or "2021-01-01")  # `2021-01-01` is the default value
```

Comment:
The same question as below. 
```suggestion
        pendulum.parse(config.get("start_date", "2021-01-01"))  # `2021-01-01` is the default value
```